package.json

```diff
@@ -803,6 +794,48 @@
 				"command": "openshift.sandbox.signup",
 				"title": "Provision OpenShift Developer Sandbox",
 				"category": "OpenShift"
+			},
+			{
+				"command": "openshift.Serverless.openFunction",
+				"title": "Open Function",
```

Comment:
@mkuznyetsov What does Open Function command do ?

Reply:
It opens the `func.yaml` in editor